services/QuillLMS/app/controllers/blog_posts_controller.rb

```diff
@@ -62,6 +62,12 @@ def search
     render 'index'
   end
 
+  def featured_blog_post
+    blog_post = BlogPost.find(params[:id])
```

Comment:
we already have the variable `@blog_post` in this controller. You might want to just re-use that variable here so we don't have two variables with such similar names.